server/webapp/WEB-INF/vm/auth/login.vm

```diff
@@ -20,7 +20,7 @@
 <head>
   <meta http-equiv="content-type" content="text/html; charset=UTF-8"></meta>
 
-  <link href="$req.getContextPath()/$concatenatedCruiseIconFilePath?#include("admin/admin_version.txt.vm")"
+  <link href="$req.getContextPath()/$concatenatedCruiseIconFilePath?${currentGoCDVersion.fullVersion()}"
```

Comment:
Intentional to bust cache :) Rails does this nicely for us by adding a checksum of the resource in the URL.